src/parse-payload.ts

```diff
@@ -17,6 +17,9 @@ export const ParsePayload = async (
 
     return payload;
   } catch (e) {
-    throw new Error("Error parsing function invocation payload", e);
```

Comment:
QQ: Is `e` always caught as an error? I think so, but also don't want to hide this error if not...

Reply:
In Deno 2 typescript seems to believe the caught object is `unknown` rather then `Error`, this is why we need to add this logic